tests/sentry/api/endpoints/test_organization_join_request.py

```diff
@@ -132,13 +132,11 @@ def test_request_to_join_email(self, mock_record):
             "join_request.created", member_id=join_request.id, organization_id=self.organization.id
         )
 
-        assert len(mail.outbox) == 2
-
-        assert mail.outbox[0].to == ["manager@localhost"]
-        assert mail.outbox[1].to == ["owner@localhost"]
-
         expected_subject = f"Access request to {self.organization.name}"
-        assert mail.outbox[0].subject == expected_subject
+        assert len(mail.outbox) == 2
+        for i in range(len(mail.outbox)):
+            assert mail.outbox[i].to in ([user.email] for user in (user1, user2))
+            assert mail.outbox[i].subject == expected_subject
```

Comment:
I think it'd be a little more explicit with a named list/tuple:

```python
users_able_to_approve_requests = [user1, user2]
assert len(mail.outbox) == len(users_able_to_approve_requests)
    for i in range(len(mail.outbox)):
        assert mail.outbox[i].to in ([user.email] for user in users_able_to_approve_requests)
        assert mail.outbox[i].subject == expected_subject
```

Just my two-cents that it wasn't immediately obvious why only two emails should be going out, and why it's those users.